.tools/readme.py

```diff
@@ -0,0 +1,32 @@
+"""Update README.md, used by pre-commit."""
+
+import re
+from pathlib import Path
+
+PROJECT_ROOT = Path(__file__).parent.parent
+
+
+def main():
+    with open(PROJECT_ROOT / "README.md") as fh:
+        readme = fh.read()
+
+    with open(PROJECT_ROOT / "src/changelist/default_config.toml") as fh:
```

Comment:
nitpick: does windows like this? Or we shouldn't be bothered anyway?

Reply:
Do you mean the "/" vs "\\" thing? I think abstracting that away is one of the ideas behind pathlib but I haven't used in much on a Windows system to be honest.